etl/steps/data/garden/gapminder/2023-09-18/under_five_mortality.meta.yml

```diff
@@ -0,0 +1,114 @@
+tables:
+  under_five_mortality:
+    title: Child Mortality Rate (Gapminder)
+    description: |
+      "This data has been compiled by Klara Johansson and Mattias Lindgren (Gapminder) from a selection of sources:
```

Comment:
This description should rather be citation_full. If it refers only to one of the origins, then it should be defined at the snapshot.